hardware_enablement/about-hardware-enablement.adoc

```diff
@@ -0,0 +1,12 @@
+[id="about-hardware-enablement"]
+= About hardware enablement on OpenShift
+include::modules/common-attributes.adoc[]
+:context: about-hardware-enablement
+
+toc::[]
+
+Many applications require specialized hardware or software which depend on kernel modules or drivers. The recommended approach to load out-of-tree kernel modules on {op-system-first} nodes is to use driver containers. To deploy  out-of-tree drivers at the same time as cluster installation (day-1) the kmods-via-containers framework can be used. For loading drivers or kernel modules on an existing {product-title} cluster (day-2) {product-title} offers several tools:
```

Comment:
s/To deploy  out-of-tree/To deploy out-of-tree/

Reply:
This one took me a second to see, lol. Fixed now thanks.